src/yarp.c

```diff
@@ -5532,6 +5532,7 @@ parse_block(yp_parser_t *parser) {
   if (accept(parser, YP_TOKEN_PIPE)) {
     parameters = parse_block_parameters(parser);
     parser->command_start = true;
+    while (accept(parser, YP_TOKEN_NEWLINE));
```

Comment:
There should only be one here, the rest should be ignored newlines. So this can just be `accept`.